README.md

```diff
@@ -64,7 +64,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 **Open the live-demo of the project**
-[Project Link][]
+[Project Link][https://theresetuyi.github.io/Leaderboard/]
```

Comment:
- [ ] You have added a demo link that does not lead to the page deployment. Kindly remove this link or mark it as N/A or ensure it leads to the correct page of the leaderboard deployment. This will help us to make the readme file more professional ☺️ .